docker/dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:latest
+
+RUN apt-get update && apt-get upgrade -y && \
+    apt-get install -y nodejs 
+
+RUN mkdir -p /usr/src/app
+
+WORKDIR /usr/src/app
+
+COPY package.json /usr/src/app
+
+RUN npm install
+
+COPY . /usr/src/app
+
+EXPOSE 7500
+
+CMD ["npm","start"]
```

Comment:
The npm start command uses ts-node to start the server, which according to ts-node itself, should be used only for development. Again, you can make a separate `Dockerfile` for the production image, where the code is first compiled using npm run build, and then the compiled main file is served by using node.